server/db_models/README.md

```diff
@@ -0,0 +1,21 @@
+## Database setup
+
+1. The postgreSQL server is running, and the database you want to use is ready.
+2. Environment variable DB_CONNECTION_STRING is set to the connect string for your postgres server. See default string in config.py for an example.
+3. Environment variable FLASK_APP is set to access app.py. 
+    - Other environment variables are set appropriately.
+4. (optional) Do a check with previous basic routes: 
+    - start the app (`flask run`)
+    - localhost:5000/v1/welcome
+    - `curl -X POST -H 'Content-Type: application/json' -d '{"teamName": "sally"}' http://localhost:5000/v1/ping`
+    - stop the app
+5. Ensure FLASK-SQLAlchemy, FLASK-Migrate and psycopg2 are installed in your virtual environment.
+6. Open a terminal and go to the server directory. Type `flask db init`. This will create a migrations directory. You only need to do this the **first time** using FLASK-Migrate for a database.
+7. Type `flask db migrate`. FLASK-Migrate will check the database against your models, and generate scripts (in versions directory) to upgrade the database (and to undo the actions with downgrade).
```

Comment:
It seems that these scripts were already generated by you in this PR. It's probably worth mentioning this in the README, as rerunning these commands might overwrite our migrations directories and make them out of sync.

Reply:
Did you give it a go? I'm not sure if there will be over-writing.
Anyhow, I had a debate about this with myself. The official docs for FLASK-Migrate advocates that the migrations directory be under version control; the given use case is to sync development and production dbs. However, I'm unclear how this is beneficial at the prototype stage when we're working on separate dbs. I'm leaning towards excluding migrations, as it could be a source of unnecessary confusion. We can just generate the necessary scripts locally within our work space from the current models in the checked-out branch we're working on.

Reply:
@rice-cake I agree with you that generating the local scripts sounds the best. From how I understand, "flask db migrate" will auto-generate scripts based on the model and db. So as long as we migrate the db whenever we merge to dev, things should work out.

Reply:
what do you mean by "we migrate the db whenever we merge to dev"? which db? should we stamp all our dbs with the same alembic version, and merge the scripts generated by Migrate to dev? I think there are dependencies between scripts as well.

Reply:
I think we should each have our local alembic version, and thus our own local upgrade scripts. I don't think we should include the scripts in the dev branch, and like you said exclude migrations.